aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -599,7 +609,6 @@ public static ModifyDbInstanceRequest modifyDbInstanceAfterCreateRequest(final R
                 .engineVersion(model.getEngineVersion())
                 .manageMasterUserPassword(model.getManageMasterUserPassword())
                 .masterUserPassword(model.getMasterUserPassword())
-                .masterUserPassword(model.getMasterUserPassword())
```

Comment:
Can you elaborate why `masterUserPassword` is being removed here?

Reply:
it's duplicated on line 611. not sure it was duplicate.

Reply:
Apology missed it :)